### PR TITLE
[release-3.9] Fix version issue in openshift_facts when using custom image.

### PR DESCRIPTION
**Pull request by patrickdillon**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1634004

This fixes a bug where customers using internal registry mirrors and custom images would fail to have the version added to openshift facts by passing the openshift_cli_image variable to openshift_facts. The openshift_cli_image will use a custom image or the default images; before it only used the default images, which might not be available in this case with an internal registry.

### CODE REVIEW

File: roles/openshift_facts/library/openshift_facts.py [hunk 1]
```diff
@@ -866,7 +864,13 @@ def get_openshift_version(facts):
         image_tag = get_container_openshift_version(deployment_type)
         if image_tag is None:
             return version
-        cli_image = image_type_dict[deployment_type] + ":" + image_tag
+
+        try:
```
Comment (michaelgugino) [inline roles/openshift_facts/library/openshift_facts.py]:
```python
openshift_cli_image = facts['common'].get('openshift_cli_image')
if not openshift_cli_image:
    return version
```

Comment (patrickdillon) [reply roles/openshift_facts/library/openshift_facts.py]: Done, if you want to take another look.

**Pull request merged**